extensions/wikia/EditAccount/SpecialEditAccount_body.php

```diff
@@ -602,4 +610,51 @@ private function logOut() {
 
 		return $ok;
 	}
+
+	private function addFanContributor() {
+		global $wgWordpressAPIUser, $wgWordpressAPIPassword;
+
+		$url = 'http://sandbox2.fandom.wikia.com/ajax/create_user/';
+		$fields = [
+			'wp_login' => $this->mUser->getName(),
+			'wp_email' => $this->mUser->getEmail(),
+			'fandom_id' => $this->mUser->getId(),
+		];
+
+		$ch = curl_init();
+
+		curl_setopt( $ch, CURLOPT_URL, $url );
+		curl_setopt( $ch, CURLOPT_USERPWD, $wgWordpressAPIUser . ":" . $wgWordpressAPIPassword );
+		curl_setopt( $ch, CURLOPT_POSTFIELDS, $fields );
+		curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+
+		$result = json_decode( curl_exec( $ch ), true );
+		curl_close( $ch );
+
+		if (
+			$result === false ||
+			empty( $result['success'] ) ||
+			empty( $result['data']['wp_user_id'] )
+		) {
+			if ( !empty( $result['data'][0]['message'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "message" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)